README.md

```diff
@@ -0,0 +1,179 @@
+|   |
++---data_in                           <- Data downloaded from external sources can be saved here. Files in here will not be committed
+|   |   .gitkeep                      <- This is a placeholder file that enables the otherwise empty directory to be committed
+|   |
++---data_out                          <- Any data saved as files will be stored here. Files in here will not be committed
+|   |   .gitkeep                      <- This is a placeholder file that enables the otherwise empty directory to be committed
+|   |
++---renv                              <- Files used by renv, the dependency management package.
+|   |   .gitignore                    <- Auto generated by renv. Prevents packages from being committed to your git repository
+|   |   activate.R                    <- Auto generated by renv to activate its package management features. Should not be edited
+|   |   settings.json                 <- Auto generated by renv. Advanced users can adjust renv settings here
+|   |
+|   +---library                       <- renv will install the packages needed by the process here
+|   |
++---src                               <- Scripts with functions for use in 'create_publication.r'. Contains the project's codebase
+|   |
+|   +---data_exports
+|   |       .gitkeep                  <- This is a placeholder file that enables the otherwise empty directory to be committed
+|   |
+|   +---data_ingestion                <- functions to import and preprocess data
+|   |       .gitkeep                  <- This is a placeholder file that enables the otherwise empty directory to be committed
+|   |       import_data.r             <- Gets data from external sources
+|   |
+|   +---data_processing               <- functions to process data i.e. clean and derive new fields, create aggregations etc.
+|   |       .gitkeep                  <- This is a placeholder file that enables the otherwise empty directory to be committed
+|   |       aggregations.r            <- Functions that create the aggregate counts needed in the outputs
+|   | 
+|   +---utils                         <- utility functions not captured by the folder above
+|   |       .gitkeep                  <- This is a placeholder file that enables the otherwise empty directory to be committed
+```
+
+### `root`
+
+In the highest level of this repository (known as the 'root'), the key file is: `create_report.r` (see above for more info). The `create_report.r` file is where people will go to actually run the pipeline.
+
+### `src`
+
+This directory contains the meaty parts of the code. By organising the code into logical sections, we make it easier to understand, maintain and test. Moreover, tucking the complex code out of the way means that users don't need to understand everything about the code all at once.
+
+### `tests`
```

Comment:
There's no tests folder included in the project structure above, are you planning to add when tests are added?

Reply:
ah yeh I meant to remove all mentions of tests at this point, thanks for spotting!